test/functional/feature_mine_cached.py

```diff
@@ -46,19 +46,9 @@ def run_test(self):
 
         to = {}
         wallet1_addr1 = self.nodes[1].getnewaddress("", "legacy")
-        to[wallet1_addr1] = ["10@" + token0_symbol]
+        to[wallet1_addr1] = ["10@" + token0_symbol, "10@0"]
 
         assert_raises_rpc_error(-5, None, self.nodes[0].sendtokenstoaddress, {}, to)
 
-        self.nodes[0].importprivkey(self.nodes[1].dumpprivkey(wallet1_addr1))
-
-        self.nodes[0].sendtokenstoaddress({}, to)
-
-        self.nodes[0].unloadwallet('')
-        self.nodes[0].loadwallet('')
-
-        self.nodes[0].sendtokenstoaddress({}, to)
-
-
 if __name__ == '__main__':
```

Comment:
Why it's removed?

Reply:
You can fix that later I'm just doing a quick patch for @uzyn the basic token functionality is already handled by many other tests so it's not needed here